cpu/cc2538/radio/cc2538_rf.c

```diff
@@ -65,9 +65,9 @@ static const init_pair_t init_table[] = {
     {&RFCORE_XREG_FRMCTRL1,  0x00                     },
     {&RFCORE_XREG_SRCMATCH,  0x00                     },
     {&RFCORE_XREG_FIFOPCTRL, CC2538_RF_MAX_DATA_LEN   },
-    {&RFCORE_XREG_RFIRQM0,   FIFOP | RXPKTDONE        },
+    {&RFCORE_XREG_RFIRQM0,   FIFOP | RXPKTDONE | SFD  },
```

Comment:
```suggestion
#if IS_USED(MODULE_IEEE802154_RADIO_HAL)
    {&RFCORE_XREG_RFIRQM0,   FIFOP | RXPKTDONE | SFD  },
#else
    {&RFCORE_XREG_RFIRQM0,   FIFOP | RXPKTDONE        },
#endif
```

Netdev does not use this yey

Reply:
sure!